src/css/layout/ScrollbarLayout.css

```diff
@@ -42,7 +42,6 @@
   overflow: hidden;
   position: absolute;
   z-index: 1;
-  transition-duration: 250ms;
```

Comment:
Nice job tracking this down!  I think the issue is that this duration is intended for the width transition, but not the position.  Rather than removing this, could you modify the transition-property field and remove background-color & position so it's just width?

I don't think we want any transition for position, and the background-color is handled by the after element and it's CSS below.

Reply:
Done